components/popover/stories/template.js

```diff
@@ -179,19 +180,21 @@ export const Template = ({
 	`;
 };
 
-export const Variants = (args, context) => {
+/**
+ * Template that displays a Popover with every value of the "position" option.
+ */
+export const TipPlacementVariants = (args, context) => {
 	const placementOptions = context?.argTypes?.position?.options ?? [];
 	return html`
-		<div style=${styleMap({
-			"display": window.isChromatic() ? "none" : "contents",
-		})}>
-			${Template(args, context)}
-		</div>
-		<div style=${styleMap({
-			"display": window.isChromatic() ? "flex" : "none",
-			"flex-direction": "column",
-			"align-items": "flex-start",
-		})} class="spectrum-Typography">
+		<div
+			style=${styleMap({
+				"display": "grid",
+				"gap": "16px",
+				"grid-template-columns": "repeat(auto-fit, minmax(232px, 1fr))",
+				"max-width": "1000px",
+			})} 
+			class="spectrum-Typography"
```

Comment:
```suggestion
			class="spectrum-Typography"
```

I don't recommend invoking the spectrum-Typography class manually; this should come from the `Typography` template, but I'm also not sure it's valuable to add it here either.

Reply:
This is removed.